### DISCUSSION TITLE
when does mod take effect

### DISCUSSION THREAD

**Discussion by augustusexe** [Monika-After-Story/MonikaModDev#9047] [general]
Do you have to get to spaceroom part for mod to work? how does it activate and how do you know when its there? i installed it but i dont see any difference although im already at spaceroom part. what should i do?

**Comment by lunulae** [comment]
You can either install it while you're at the spaceroom chapter or after. (or before, but it will assume you completed the game if you aren't at the spaceroom chapter.) 
There will be intro dialogue either way talking about you installing a mod, so if you haven't gotten that you might need to troubleshoot your install.
